packages/components/src/components/button/button.jsx

```diff
@@ -8,6 +8,7 @@ const Button = ({
     children,
     className = '',
     classNameSpan,
+    circular,
```

Comment:
Please ignore my previous comment, now that I think of it, `rounded` seems like a better naming.